wifiphisher/common/phishinghttp.py

```diff
@@ -157,6 +158,12 @@ def post(self):
         file_path = template_directory + render_file
         self.render(file_path, **template.get_context())
 
+        # Finds victim object that corresponds to ip addres and try to discover OS by first request
+        victims_instance = victim.Victims.getInstance()
+        for key in victims_instance.victims_dic.keys():
+            if self.request.remote_ip == victims_instance.victims_dic[key].ip_address:
+                victim_object = victims_instance.victims_dic[key]
+                victim_object.associate_victim_ip_to_os(self.request.remote_ip, self.request.full_url())
```

Comment:
The `associate_victim_ip_to_os` should be a method of `Victims` class and include the `for` loop as well.